lib/cocoapods/installer.rb

```diff
@@ -279,8 +279,18 @@ def validate_build_configurations
     #
     def clean_sandbox
       sandbox.public_headers.implode!
```

Comment:
headers are fine they dont seem to affect the re-compilation aspect, its mostly the dummy file the xcconfig and the pch